Pool/PvP.vb

```diff
@@ -807,4 +839,21 @@ Public Class PvP
     Private Sub QuitToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles QuitToolStripMenuItem.Click
         Application.Exit()
     End Sub
+
+    Private Sub cbGames_SelectedIndexChanged(sender As Object, e As EventArgs) Handles cbGames.SelectedIndexChanged
+        For Each gameEntry As DictionaryEntry In gamesHT
+            If gameEntry.Value.Equals(cbGames.SelectedItem) Then
+                game = gameEntry.Key
+                Exit For
+            End If
+        Next
```

Comment:
Run a test on this check that we are populating that variable. This is used in the selAllWins_v1 store procedure multiple times.

Reply:
Fix selAllWins to the new version on lines 718/796.

Reply:
Fixed where the new store procedure is a need. Name is editing on both tables and deleting unregisters which can then be deleted as unregistered. 
Keeping this for now as a complete deletion would need to be done only after unregistering if that name is tied to a Login.